speech-to-text/src/main/java/com/ibm/watson/speech_to_text/v1/model/RecognizeOptions.java

```diff
@@ -101,17 +125,16 @@
     String PT_BR_BROADBANDMODEL = "pt-BR_BroadbandModel";
     /** pt-BR_NarrowbandModel. */
     String PT_BR_NARROWBANDMODEL = "pt-BR_NarrowbandModel";
+    /** pt-BR_Telephony. */
+    String PT_BR_TELEPHONY = "pt-BR_Telephony";
     /** zh-CN_BroadbandModel. */
     String ZH_CN_BROADBANDMODEL = "zh-CN_BroadbandModel";
     /** zh-CN_NarrowbandModel. */
     String ZH_CN_NARROWBANDMODEL = "zh-CN_NarrowbandModel";
   }
 
   protected InputStream audio;
-
```

Comment:
Make sure this is not a hand edit being deleted. Why was this serialized before?